shopfloor_vendor_packaging/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import models
+from . import actions
```

Comment:
from .
```suggestion
from . import actions
from . import models
```
you would've noticed if there were tests :)